knotx-example/knotx-mocks/src/main/java/com/cognifide/knotx/mocks/service/MockRemoteRepositoryHandler.java

```diff
@@ -70,4 +71,8 @@ private String getContentPath(String path) {
     }
   }
 
+  private String normalizeEndline(String content) {
+    return content.replace(DOS_NEWLINE, NIX_NEWLINE).replace(MAC_NEWLINE, NIX_NEWLINE);
```

Comment:
Could we just get rid of all endlines (move all code to a single line) and end it with UNIX eol?